features/deprovision.feature

```diff
@@ -8,5 +8,5 @@ Feature: Deprovisioning
 
   Scenario: Deprovision resource
     When I DELETE "/v2/service_instances/9b292a9c-af66-4797-8d98-b30801f32a77?service_id=service-id-here&plan_id=plan-id-here"
-    Then the HTTP response status code is "200"
+    Then the HTTP response status code is "410"
```

Comment:
I expected to see these tests here:
- [ ] deprovision with policy preserved (space / org layers still exist, but instance ID resource is gone)
- [ ] deprovision without policy preserved (space / org layers gone, instance ID resource is gone)

it looks like they're happening below, in the integration tests - so ok.

Reply:
based on [the spec](https://github.com/openservicebrokerapi/servicebroker/blob/v2.14/spec.md#response-9) 200 should be returned if the service instance was deleted, so I would expect to see 200 returned on successful deprovision.

it looks like 410 should only be returned if deprovision is called on a service instance that doesn't exist, fwiw

Reply:
Hmm, ok. I had first read the spec as the `410 gone` representing the "thing" that the service broker instance in that space represents. I can see how on a second reading it's referring to the service broker instance record in CF itself.

I will make the change. Thanks!